{{ cookiecutter.package_name|slugify }}/pyproject.toml

```diff
@@ -58,14 +58,14 @@ uvicorn = { extras = ["standard"], version = "^0.16.0" }
 # https://python-poetry.org/docs/master/managing-dependencies/
 # TODO: Split in `tool.poetry.group.dev` and `tool.poetry.group.test` when Poetry 1.2.0 is released.
 [tool.poetry.dev-dependencies]
-cruft = "^2.10.1"
 black = "^22.1.0"
 commitizen = "^2.20.2"
 coverage = { extras = ["toml"], version = "^6.2" }
+cruft = "^2.10.1"
 darglint = "^1.8.1"
 flake8 = "^4.0.1"
 flake8-comprehensions = "^3.7.0"
```

Comment:
Could you move this one down after `flake8-bugbear`?

Reply:
Done, moved around flake8-print as well while at it :p